plan/typeinferer.go

```diff
@@ -319,6 +319,12 @@ func (v *typeInferrer) handleFuncCallExpr(x *ast.FuncCallExpr) {
 	case "connection_id":
 		tp = types.NewFieldType(mysql.TypeLonglong)
 		tp.Flag |= mysql.UnsignedFlag
+	case "find_in_set":
+		if x.Args[0].GetType().Tp == mysql.TypeNull || x.Args[1].GetType().Tp == mysql.TypeNull {
```

Comment:
Thanks, @shenli. I did this because official mysql returned a NULL to me when I executed "select find_in_set(null,'foo')". I will check it out.